app/services/PenaltiesService.scala

```diff
@@ -67,4 +83,10 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) {
   def findTotalLSPFromPayload(payload: ETMPPayload): BigDecimal = {
     payload.penaltyPoints.map(_.financial.map(_.amountDue)).collect{ case Some(x) => x }.foldRight(BigDecimal(0))(_ + _)
   }
+
+  def estimatedVATInterest(payload: ETMPPayload): (BigDecimal, Boolean) = {
```

Comment:
Could we add a verb at the beginning here - like:
```suggestion
  def findEstimatedVATInterest(payload: ETMPPayload): (BigDecimal, Boolean) = {
```